.github/workflows/update_golang_dependencies.yml

```diff
@@ -41,13 +41,17 @@ jobs:
       - name: Create Pull Request
         uses: peter-evans/create-pull-request@v4
         with:
+          token: ${{ secrets.CREATE_PR_VITESS_BOT }}
           branch: "upgrade-go-deps-on-main"
           commit-message: "upgrade go deps"
           signoff: true
           delete-branch: true
+          team-reviewers: Release
           title: "Upgrade the Golang Dependencies"
           body: |
             This Pull Request updates all the Golang dependencies to their latest version using `go get -u ./...`.
+            
+            cc @vitessio/release
```

Comment:
Line 60 should change to `Dependencies`.

Reply:
We dont have a `Dependencies` label, but we have `dependencies`, should we use `Type: Dependencies` instead?

Reply:
Fixed via 457977d379